helm/adelphi/templates/cassandra-clusters.yaml

```diff
@@ -78,7 +75,7 @@ spec:
           serverType: cassandra
           serverVersion: {{ .Values.target.version }}
           {{- if .Values.target.git_identifier }}
-          serverImage: {{`{{inputs.parameters.registry_ip}}`}}:30000/adelphi/management-api-for-apache-cassandra
+          serverImage: localhost:30000/adelphi/management-api-for-apache-cassandra
```

Comment:
containerd only allows connections to insecure registries running on localhost, so now we have a proxy running locally on port 30000 that in turn points to the real registry running at the `docker-registry` service.
So we don't need the `registry_ip` parameter anymore, which in one way is also a simplification.